src/sentry/static/sentry/app/views/organizationGroupDetails/groupDetails.tsx

```diff
@@ -244,8 +244,8 @@ class GroupDetails extends React.Component<Props, State> {
     const isLoading = loading || (!group && !isError);
 
     return (
-      <DocumentTitle title={this.getTitle()}>
-        <React.Fragment>
+      <React.Fragment>
+        <DocumentTitle title={this.getTitle()}>
```

Comment:
Swapping these elements is likely the cause of the pre-commit failure:

```
  at DocumentTitle.Object.<anonymous>.DocumentTitle.render (node_modules/react-document-title/index.js:31:27)
```

Reply:
Yep, that was it!